assets/js/googlesitekit/widgets/components/WidgetRenderer.js

```diff
@@ -39,14 +44,15 @@ const WidgetRenderer = ( { slug } ) => {
 
 	// Capitalize the "component" variable, as it is required by JSX.
 	const { component: Component, wrapWidget } = widget;
-	const widgetComponent = <Component slug={ slug } />;
+	const widgetComponent = <Component className={ wrapWidget ? className : undefined } ref={ ref } slug={ slug } />;
 
-	return widgetComponent && wrapWidget
```

Comment:
This shouldn't be removed, if the component returns `null`, it shouldn't be wrapped. We cannot rely on the approach where `Widget` returns `null` if it has no `children`, because `Widget` is not a mandatory component here (that approach only works for the `wrapWidget` approach).